src/e2e/utils/helpers.ts

```diff
@@ -30,7 +30,7 @@ export const ENV_URLS = {
   local: "http://localhost:6060",
   heroku: "https://fx-breach-alerts.herokuapp.com",
   stage: "https://stage.firefoxmonitor.nonprod.cloudops.mozgcp.net",
-  prod: "https://monitor.firefox.com",
+  prod: "https://monitor.mozilla.org",
```

Comment:
I imagine this changed line will make our prod E2E tests fail until we've fully transitioned? (Though I'm not actually sure if they succeed today...)

Possibly it should wait until we're ready to fully transition? (Similarly for the link in `<AppPicker>`, which has not been updated yet.)

Reply:
Yes good call-out - I'm planning for this change to roll out as soon as the dependencies like DNS, TLS, and the FxA redirect URI are ready to go.